src/qt/intro.cpp

```diff
@@ -139,17 +140,25 @@ Intro::Intro(QWidget *parent, int64_t blockchain_size_gb, int64_t chain_state_si
     );
     ui->lblExplanation2->setText(ui->lblExplanation2->text().arg(PACKAGE_NAME));
 
+    const int min_prune_target = std::ceil(MIN_DISK_SPACE_FOR_BLOCK_FILES / 1e9);
+    ui->pruneGB->setRange(min_prune_target, std::numeric_limits<int>::max());
```

Comment:
77f3f2dda6918b0986f785b3061cfb8c97ed2b5c

Just `setMinimum`?

nit, `min_prune_target_gb`?

Reply:
Need to set the maximum too... Default is IIRC 100